### PR TITLE
A package is available on Alpine Linux

### PR DESCRIPTION
**Pull request by WhyNotHugo**
I haven't fully tested these changes since I'm not sure how to build the website itself. I've followed the style of existing packages so I think everything should be in order.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -178,6 +178,17 @@ instructions:
...
+        - name: Alpine Linux
+          id: alpine
+          instructions: >-
+            Packages are available from the [community respositories](https://pkgs.alpinelinux.org/packages?name=openrazer).
+
+
+                doas apk add openrazer
+
+
+            The kernel modules are built via akms.
```
Comment (z3ntu) [inline index.md]: Maybe link to upstream reference to akms since probably no-one reading this will know akms.

File: index.md [hunk 2]
```diff
@@ -178,6 +178,17 @@ instructions:
     - title: Community Supported
       summary: Packaged and supported by the wider community.
       distros:
+        - name: Alpine Linux
+          id: alpine
+          instructions: >-
+            Packages are available from the [community respositories](https://pkgs.alpinelinux.org/packages?name=openrazer).
+
+
+                doas apk add openrazer
```
Comment (z3ntu) [inline index.md]: I think `openrazer-src` also needs to be installed explicitly?

**Review by z3ntu — CHANGES_REQUESTED** [review]
Yeah, for some reason I've never added it to the website..

**Comment by lah7** [comment]
Formatting looks good 👍🏼  

Just recently refactored the site code to Markdown/Frontmatter. Sorry, didn't realise the instructions were missing in the README - I'll add them in later!

File: index.md (hunk 2 above)
Comment (WhyNotHugo) [reply index.md]: True, fixed.

File: index.md (hunk 1 above)
Comment (WhyNotHugo) [reply index.md]: Done

**Review by z3ntu — APPROVED** [review]
I'd still change the commit message to e.g. "Add installation instructions for Alpine Linux"

Maybe with the message "The package has been available on the official repos for a while but was never added to the website." or something.

Also I didn't check how it looks when the website it built, someone should verify this before merging.

**Review by lah7 — APPROVED** [review]
Noticed the distro logo was missing, so I added one in line with the others. Looks good to me!

👇🏼  Will fix that `repositories` typo first

Screenshot_20230912_130228

**Comment by WhyNotHugo** [comment]
Thanks for taking care of those last details

**Pull request merged**